php/WcaOauth.php

```diff
@@ -3,11 +3,13 @@
 class WcaOauth
 {
     CONST ACCESS_TOKEN_URI = "https://www.worldcubeassociation.org/oauth/token";
+    CONST OAUTH_AUTHORIZE_URI = "https://www.worldcubeassociation.org/oauth/authorize";
     CONST USER_URI = "https://www.worldcubeassociation.org/api/v0/me";
 
     protected $applicationId;
-    protected $redirectUri;
     protected $applicationSecret;
+    protected $redirectUri;
```

Comment:
A good line move. I like having applicationId and applicationSecret next to each other as well =)